src/controllers/expenses.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const { Op } = require('sequelize');
+const expensesService = require('../services/expenses');
+
+const create = async(req, res) => {
+  try {
+    const expense = await expensesService.create(req.body);
+
+    res.statusCode = 201;
+    res.send(expense);
+  } catch (error) {
+    res.statusCode = 400;
+    res.send('Enter all fields');
+  }
+};
+
+const getAll = async(req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const filters = {};
+
+  if (userId) {
+    filters.userId = userId;
+  }
+
+  if (categories) {
+    filters.category = categories;
+  }
+
+  if (from && to) {
+    filters.createdAt = { [Op.between]: [from, to] };
+  }
```

Comment:
Consider using sanitizing and validation middleware for this checks (you can use lib or to write middleware function)